src/main/java/sc/fiji/bdvpg/bdv/projector/Projection.java

```diff
@@ -1,8 +1,13 @@
 package sc.fiji.bdvpg.bdv.projector;
 
-public class ProjectionTypes
+public class Projection
 {
+	public static final String PROJECTION_MODE = "Projection Mode";
+	public static final String PROJECTION_MODE_SUM = "Sum";
+	public static final String PROJECTION_MODE_AVG = "Avg";
+
 	public static final String MIXED_PROJECTOR = "Mixed Projector";
 	public static final String SUM_PROJECTOR = "Sum Projector";
```

Comment:
cf comment above, maybe change this to int to fasten the computation

Reply:
Ok. I could do this, but long term I think an enum would be the cleanest. 
The only reason I was choosing String constants was because this is the only thing that works in `Command` choices. Thus, using int's not would make it more complicated because we have to implement the constants twice.

Reply:
Ok, I'll investigate the performance and report back